dev/fattest.simplicity/src/componenttest/topology/impl/LibertyClient.java

```diff
@@ -673,26 +674,32 @@ public ProgramOutput startClientWithArgs(boolean preClean, boolean cleanStart,
         //FIPS 140-3
         // if we have FIPS 140-3 enabled, and the matched java/platform, add JVM arg
         if (isFIPS140_3EnabledAndSupported()) {
-            Log.info(c, "startClientWithArgs", "The JDK version: " + javaInfo.majorVersion() + " and vendor: " + JavaInfo.Vendor.IBM);
-
-            if (javaInfo.majorVersion() >= 11){
-                Log.info(c, "startClientWithArgs", "FIPS 140-3 global build properties is set for Client " + getClientName()
-                                                   + " with IBM Java " + javaInfo.majorVersion() + ", adding required JVM arguments to run with FIPS 140-3 enabled");
-                                                   
-                JVM_ARGS += " -Dsemeru.fips=true";
-                JVM_ARGS += " -Dsemeru.customprofile=OpenJCEPlusFIPS.FIPS140-3-Custom";
-                JVM_ARGS += " -Djava.security.properties=" + getSemeruFips140_3CustomProfileLocationAndPrintFileContents();
-                JVM_ARGS += " -Dcom.ibm.fips.mode=140-3";
-                // JVM_ARGS += " -Djavax.net.debug=all";  // Uncomment as needed for additional debugging
-            } else if (javaInfo.majorVersion() == 8) {
-                Log.info(c, "startClientWithArgs", "FIPS 140-3 global build properties is set for Client " + getClientName()
-                                               + " with IBM Java 8, adding JVM arguments -Xenablefips140-3, ...,  to run with FIPS 140-3 enabled");
-
-                JVM_ARGS += " -Xenablefips140-3";
-                JVM_ARGS += " -Dcom.ibm.jsse2.usefipsprovider=true";
-                JVM_ARGS += " -Dcom.ibm.jsse2.usefipsProviderName=IBMJCEPlusFIPS";
-                JVM_ARGS += " -Dcom.ibm.fips.mode=140-3";
-                // JVM_ARGS += " -Djavax.net.debug=all";  // Uncomment as needed for additional debugging
+            if (GLOBAL_ENHANCED_ALGO) {
+                JVM_ARGS += "-Duse.enhanced.security.algorithms=true";
+                JVM_ARGS += "-Dcom.ibm.ws.beta.edition=true";
```

Comment:
```suggestion
                JVM_ARGS += " -Duse.enhanced.security.algorithms=true";
                JVM_ARGS += " -Dcom.ibm.ws.beta.edition=true";
```

these need a space before them, otherwise it'll be 
```
$JVM_ARGS-Duse.enhanced.security.algorithms=true-Dcom.ibm.ws.beta.edition=true
```